corehq/apps/reports/templates/reports/form/edit_submission.html

```diff
@@ -2,9 +2,11 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 
+{% block js-inline %}{{ block.super }}
+{% include 'cloudcare/includes/touchforms-inline.html' %}
+{% endblock %}
```

Comment:
it seems this approach was already tried once: https://github.com/dimagi/commcare-hq/pull/9628/files#r47959495 @biyeun @benrudolph do you know if this is still going to be a problem? there isn't really any other great way to fix this